index.js

```diff
@@ -0,0 +1,123 @@
+webix.ready(function () {
+  var head = {
+    view: "toolbar",
+    css: "header",
+    id: "myToolbar",
+    height: 50,
+    padding: 10,
+    cols: [
+      { view: "label", label: "My App", css: "logo" },
+      {
+        view: "button",
+        label: "Profile",
+        type: "icon",
+        icon: "wxi-user",
+        width: 150,
+        css: "profile-icon",
+      },
+    ],
+  };
+
+  var sideBarData = [
+    { id: 1, value: "Dashboard", $css: "sidebar-item" },
+    { id: 2, value: "Users", $css: "sidebar-item" },
+    { id: 3, value: "Products", $css: "sidebar-item" },
+    { id: 4, value: "Locations", $css: "sidebar-item" },
+  ];
+
+  var sideBar = {
+    view: "list",
+    css: "side-bar",
+    width: 250,
+    scroll: false,
+    data: sideBarData,
+  };
+
+  var connectStatus = {
+    css: "connect-status",
+    height: 50,
+    template: "<span class='webix_icon wxi-check'></span> Connected",
+  };
+
+  var filmsTable = {
+    view: "datatable",
+    autoConfig: true,
+    data: small_film_set,
+  };
+
+  var formButtons = [
+    {
+      view: "button",
+      type: "form",
+      value: "Add new",
+      css: "form-button",
+    },
+    {
+      view: "button",
+      type: "form",
+      value: "Clear",
+    },
+  ];
+
+  var filmsForm = {
+    view: "form",
+    borderless: true,
```

Comment:
I wouldn't add `borderless` there, since it against the design (see the screenshot in the task) and looks strange near other borders:
![image](https://github.com/user-attachments/assets/00593739-49e2-453c-9824-00c9b470d020)

I assume that this was done in an attempt to render "Edit films" on top of the form without extra borders. 
But there's an easier solution (see line 24): https://snippet.webix.com/chrbters
https://docs.webix.com/desktop__template.html#templatetypes (see "Section")
Content of the form is borderless by default, and the form can include not only inputs/buttons, but any suitable views.